store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -475,7 +475,7 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
     // re-create rollup metadata
     for { rollup <- rollups } RollupCreatedOrUpdatedHandler(pgu, postUpdateTruthCopyInfo, rollup)
     // re-create rollup tables
-    updateRollups(pgu, postUpdateTruthCopyInfo, dropFirst = false)
```

Comment:
The time spent was mainly to investigate and repro the bug, which is summarized here. https://socrata.atlassian.net/browse/EN-4519